pkg/resources/nifi/secretconfig.go

```diff
@@ -130,6 +130,20 @@ func (r *Reconciler) getNifiPropertiesConfigString(nConfig *v1.NodeConfig, id in
 		webProxyHosts = strings.Join(append(dnsNames, base.WebProxyHosts...), ",")
 	}
 
+	if strings.Contains(config.NifiPropertiesTemplate, "user.oidc.client.secret") {
+		// read secret value
+		clientSecret := &corev1.EnvVarSource{
+				SecretKeyRef: &corev1.SecretKeySelector{
+					LocalObjectReference: corev1.LocalObjectReference{
+						Name: OidcConfiguration.SecretRef.Name,
+						NameSpace: OidcConfiguration.SecretRef.NameSpace,
+					},
+					Key: clientSecret,
```

Comment:
Can you please document (ad nauseum) what the key in the secret should be for the referenced secret? This should be documented on the website documentation, in `nificluster_types.go`, and in any comment where is makes sense. This way, it's very clear what the secret should contain.